scripts/pysa_fuzzer/build_script.py

```diff
@@ -0,0 +1,123 @@
+import subprocess

```

Comment:
naming: I would call this script `run.py` and `fuzzerX.py` would be called `code_generator.py`